src/python/pants/notes/2.16.x.md

```diff
@@ -130,7 +130,7 @@ Enable the `pants.backend.experimental.cue` backend to add this support.
 
 * Ugrade Pex to 2.1.120 ([#17957](https://github.com/pantsbuild/pants/pull/17957))
 
-* Adds `workdir` field for `experimental_shell_command` and friends ([#17928](https://github.com/pantsbuild/pants/pull/17928))
+* Adds `workdir` field for `shell_command` and friends ([#17928](https://github.com/pantsbuild/pants/pull/17928))
```

Comment:
Not sure if it's a good idea to change name for already released versions, as the new name is not available in that context.

Reply:
I don't believe we've even branched 2.16 yet. To be clear, in that context, are you suggesting that I should keep the old target name for the release notes?

Reply:
yes for the notes referencing dev releases, they are out there and usable. So if I put `2.16.0.dev5` into my `pants.toml` I'll have the old `esc` target not this new one. So let the notes reflect accurately the name usable at the version the note refers to is my suggestion :)

Reply:
Hmm, ok, that's fair enough.